electrum/lnchannel.py

```diff
@@ -525,6 +546,9 @@ def is_static_remotekey_enabled(self) -> bool:
         # their local config is not static)
         return False
 
+    def has_anchors(self) -> Optional[bool]:
+        return None
```

Comment:
is that method used? 
in `create_sweeptx_their_backup_ctx` you are using `lnutil.ctx_has_anchors()` instead

Reply:
It's not being used, but it's an `abstractmethod`, so has to be implemented. With the `None` return I wanted to indicate that we don't know about the channel having anchors.